src/mr-importer/loader.cpp

```
@@ -279,6 +279,69 @@ static bool get_attribute_for_all_points(fastgltf::ComponentType component_type,
   return decode_result;
 }
 
+// Compute bounding sphere using Ritter's algorithm
+static BoundingSphere calculate_bounding_sphere(const PositionArray &positions)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)